### PR TITLE
updated blacklist

### PR DESCRIPTION
**Pull request by Ester800**
ZD #651914

### CODE REVIEW

**Review by AlexHerman1 — APPROVED** [review]
LGTM.

File: src/config.json [hunk 1]
```diff
@@ -937,8 +937,10 @@
     "metapass.fyi"
   ],
   "blacklist": [
+    "restoremetadata.com",
+    "mywalletauthenticator.weebly.com",
     "sandboxgame.app",
-    "shibasociaiclub.com",
+    "shibasocialclub.com",
```
Comment (Gudahtt) [inline src/config.json:943]: @AlexHerman1 @Ester800 Was this a mistake? This change seems unrelated to the linked ZenDesk ticket. Also we got a report that this site was legitimate: #6337

Comment (AlexHerman1) [reply src/config.json:943]: i believe the line that was removed is the phishing site, and the other is the legitimate site: shibasocialclub.com

Comment (Gudahtt) [reply src/config.json:943]: Fixed here: https://github.com/MetaMask/eth-phishing-detect/pull/6396
